app-embed-example/setup/metabase-setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+echo "seting up $1"
+# get deps
+apk add curl jq
+# get the wait-until script
+curl -L https://raw.githubusercontent.com/nickjj/wait-until/v0.2.0/wait-until -o /usr/local/bin/wait-until && \
+chmod +x /usr/local/bin/wait-until
+# run the script and everything else
+wait-until "echo 'Checking if Metabase is ready' && curl -s http://$1/api/health | grep -ioE 'ok'" 60 && \
```

Comment:
60 seconds were not enough on my machine (macbook pro M1), it did work with 120.

I also suggest adding `--connect-timeout 5 --max-time 5` to `curl`.

Reply:
I prepared Metabase inside a Mac M1 compatible container, so this shouldn't happen again